vignettes/06-summary-plot.Rmd

```diff
@@ -122,11 +122,11 @@ summarized_panicles %>%
   scale_fill_viridis_c() +
   labs(x = "Rank along primary branch",
        y = "Rank along the rachis",
-       fill = "How many panicles\nhave that node?",
-       caption = "The numbers indicate the average numbers of nodes into the tertiary branch")
+       fill = "Spikelet Number per Node",
```

Comment:
Hi Helen, 

Sorry, I think that this is imprecise. Since this plot is a summary of many panicles, the colours tells actually how many panicles have something at that position. 

For example, on the first node of the first primary branch, the colour might be yellow, because all panicles have something there, while at the 8th node of the 10th primary branch, maybe the colour is blue, because only one panicle has something there.